app/components/Calendar/Calendar.jsx

```diff
@@ -0,0 +1,39 @@
+"use strict"
+import dom from '../../utils/dom'
+import {log} from '../../utils/logger'
+import ToggleCalendar from 'ToggleCalendars'
+
+import {renderCalendar,populateGoogleDates} from '../../utils/calendar-utils'
+
+
+
+/**
+ * Calendar Component
+ * @param  {array} calendarEvents - The events for calendar to display
+ * @return {VNode}
+ */
+export default ({state: {sources,calendarEvents,route}}) => {
+console.log(sources);
+  return (
+      <section>
+        <ToggleCalendar state={sources}></ToggleCalendar>
+        <div id="calendar" onload={renderCal(sources)} />
```

Comment:
`renderCal(sources)` can be moved out of onload, because it's just being called inline anyways. I don't think we have an onload handler anyways. So even if it was written as `onload={() => rederCal(sources)}` it wouldn't get called.